File: 2089-maximum-matrix-sum/maximum-matrix-sum.java
class Solution {
    public long maxMatrixSum(int[][] matrix) {
        int min=Integer.MAX_VALUE;
        long total=0;
        int count=0;
        for(int i[]:matrix){
            for(int j:i){
                total+=Math.abs(j);
                if(j<0) count++;
                min=Math.min(min,Math.abs(j));
            }
        }
        return count%2==0?total:total-2*min;
    }
}

File: 2089-maximum-matrix-sum/maximum-matrix-sum.js
/**
 * @param {number[][]} matrix
 * @return {number}
 */
var maxMatrixSum = function(matrix) {
    let min=Number.MAX_VALUE;
    let count=0;
    let t=0;
    for(let i of matrix){
        for(let j of i){
            t+=Math.abs(j);
            if(j<0) count++;
            min=Math.min(min,Math.abs(j));
        }
    }
    return count%2==0?t:t-2*min;
};